src/imu/imu.h

```diff
@@ -34,11 +34,10 @@
 * | robot              | string             | -     | -             | Yes      | The name of the robot. | e.g. icub |
 * | model              | string             | -     | -             | Yes      | The name of the robot model. | e.g. model.urdf |
 * | port               | string             | -     | -             | Yes      | The name of the port streaming IMU data. | e.g. /icub/head/inertials |
-* | part               | string             | -     | -             | Yes      | The name of the robot part on which the sensor is mounted. | e.g. head |
-* | controlboards      | vector of string   | -     | -             | Yes      | The list of the controlboards to open. | e.g. ("torso", "head") |
-* | sensor             | string             | -     | -             | Yes      | The name of the sensor to be tested. | e.g. head_imu_0 |
-* | frame              | string             | -     | -             | Yes      | The name of the frame on which the sensor is attached. | e.g. head_imu_0|
-* | mean_error         | double             | -     | -             | Yes      | The tolerance on the mean of the error. | |
+* | remoteControlBoards| vector of string   | -     | -             | Yes      | The list of the controlboards to open. | e.g. ("torso", "head") |
+* | axesNames          | vector of string   | -     | -             | Yes      | The list of the controlled joints. | e.g. ("torso_pitch", "torso_roll", "torso_yaw", "neck_pitch", "neck_roll", "neck_yaw") |
+* | testList           | vector of string   | -     | -             | Yes      | The list of the sensors to be tested | e.g. ("head_imu_0", "l_arm_ft") |
```

Comment:
Nitpick: in order to make more clear that in that list the user has to specify the sensor names maybe `sensorsList` is more appropriate?

Reply:
You're right! Done in https://github.com/robotology/icub-tests/pull/69/commits/9f3fc77cd85aa5f9bcb52eb65b8b53e0a594148b